ocp_resources/image_stream.py

```diff
@@ -7,3 +8,51 @@ class ImageStream(NamespacedResource):
     """
 
     api_group = NamespacedResource.ApiGroup.IMAGE_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        image_repository=None,
+        scheduled=None,
+        tag=None,
+        teardown=True,
+        privileged_client=None,
+        yaml_file=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        **kwargs,
+    ):
+        super().__init__(
+            name=name,
+            namespace=namespace,
+            client=client,
+            teardown=teardown,
+            privileged_client=privileged_client,
+            yaml_file=yaml_file,
+            delete_timeout=delete_timeout,
+            **kwargs,
+        )
+        self.image_repository = image_repository
+        self.scheduled = scheduled
+        self.tag = tag
+
+    def to_dict(self):
+        res = super().to_dict()
+        if self.yaml_file:
+            return res
+        res.update(
+            {
+                "spec": {
+                    "lookupPolicy": {"local": False},
```

Comment:
lookupPolicy.local should be configurble; you can set the default to False

Reply:
Fixed